DEVELOPING.md

```diff
@@ -39,9 +39,12 @@ $ ${ANDROID_HOME}/tools/bin/sdkmanager \
       "patcher;v4" \
       "extras;google;m2repository" \
       "extras;m2repository;com;android;support;constraint;constraint-layout;1.0.2" \
-      "ndk;22.0.7026061"\
+      "ndk;21.4.7075529"\
```

Comment:
Why downgrade the ndk?

Reply:
It's the NDK version we use in our CI to build our C++ code, we should be consistent in this doc.

Reply:
In order to align what we use in our ci, which is [r21e](https://github.com/android/ndk/wiki/Unsupported-Downloads#r21e).